src/include/sof/audio/drc/drc.h

```diff
@@ -70,7 +70,7 @@ struct drc_comp_data {
 	struct comp_data_blob_handler *model_handler;
 	struct sof_drc_config *config;      /**< pointer to setup blob */
 	bool config_ready;                  /**< set when fully received */
-	enum sof_ipc_frame source_format;   /**< source frame format */
+	enum sof_ipc_frame sink_format;     /**< source frame format */
```

Comment:
comment update

also how do we know what we will get if the format is not set for source?

Reply:
I got confirm from @ranj063 that update both sink and source should be done as it is now. I can then remove the source -> sink commit that has this comment mistake.